lte/gateway/fabfile.py

```diff
@@ -284,11 +285,16 @@ def integ_test(
     else:
         ansible_setup(gateway_host, "dev", "magma_dev.yml")
         gateway_ip = gateway_host.split('@')[1].split(':')[0]
+    print("bbuild is set to: %s" % build)
```

Comment:
maybe move this inside the if and only print when we are building from a hash. Otherwise it is the regular flow of building from source

Reply:
In my opinion, this would be more readable this way.

```
    if github_hash:
	print("Building from github hash:  %s" % github_hash)
	execute(_from_registry)
   else:
        execute(_dist_upgrade)
        execute(_build_magma)
        execute(_run_sudo_python_unit_tests)
        execute(_start_gateway)
```